### DISCUSSION TITLE
Install guide?

### DISCUSSION THREAD

**Discussion by SteelHotdogs** [UnlegitSenpaii/FAE_Linux#19] [Q&A] [answered]
I need help installing this on linux mint.

**Comment by UnlegitSenpaii** [comment]
okay

**Reply by SteelHotdogs** [reply, marked as answer]
thx for the help

**Comment by UnlegitSenpaii** [comment]
i broke down the usage and compile guidelines into their single steps. i dont know how much more you need. if youre stuck somewhere let me know.
